core/meta/src/TProtoClass.cxx

```diff
@@ -175,7 +175,7 @@ Bool_t TProtoClass::FillTClass(TClass* cl) {
             Info("FillTClass", "Returning w/o doing anything. %s is a STL collection.",cl->GetName());
          return kFALSE;
       }
-      if (cl->Property() & kIsNamespace) {
+      if (cl->fProperty != (-1) && cl->fProperty & kIsNamespace) {
```

Comment:
Is there somewhere a documentation of the meaning of the possible `fProperty` values?

Reply:
`0` isn't a thing for `fProperty`, and isn't tested for in the original code or Philippe's modification. `fProperty & kIsWhatever` checks whether the class is a whatever. `-1` is the default value of `fProperty`, stating "we have not determined the class's properties yet" (i.e. `Property()` has not been called yet)

Reply:
`-1` means unset.  0 would means all properties was found to be 'false'.
Yes, I need to update the documentation :)